deepseq.cabal

```diff
@@ -53,6 +53,11 @@ library
     if impl(ghc < 7.6)
       build-depends: ghc-prim == 0.2.*
 
+  if impl(ghc >= 7.8)
+    other-extensions:
+      ScopedTypeVariables
+      Trustworthy
```

Comment:
Oops, I misread the phrase `other-extensions`.

Hm, the current full list of `other-extensions` is a bit of a lie since some extensions are only enabled conditionally (e.g., `DeriveGeneric`), but since it's only informative, that doesn't bother me too much. You could probably just merge these two entries with the others for simplicity's sake.